editor/src/components/canvas/canvas-strategies/strategies/resize-grid-strategy.ts

```diff
@@ -189,10 +191,9 @@ export const resizeGridStrategy: CanvasStrategyFactory = (
           ),
           propertyValueAsString,
         ),
-        setElementsToRerenderCommand([gridPath]),
       ]
 
-      return strategyApplicationResult(commands)
+      return strategyApplicationResult(commands, 'rerender-all-elements')
```

Comment:
this looks like this should be `[gridPath]`

Reply:
Fixed in 71214dfb2.